slides/src/content/unit-testing.md

```diff
@@ -332,39 +275,63 @@ beforeEach(() => {
 ```
 
 ---
-<!-- .slide: id="unit-testing-change-detection-1" -->
-## Change Detection
 
-- We must tell Angular when to run change detection during our tests
-- Use the fixture to do this:
+<!-- .slide: id="unit-testing-shallow-rendering" -->
+## Shallow Rendering
+
+- The above test fails because `AppComponent` itself uses components which we have not declared.
+- We could provide mocks for these components, or we can do a shallow render by adding the `CUSTOM-ELEMENTS-SCHEMA`
```

Comment:
1. "or we can do" => "or do"
2. is `CUSTOM-ELEMENTS-SCHEMA` hyphenated or underscored? looks in the code below like underscores